### PR TITLE
Qt: Add accelerator keys to the Receive tab

### PR DESCRIPTION
**Pull request by EchterAgo**
For easier keyboard navigation. The & in the labels is in front of the key used as accelerator, you can press Alt+Key to use the accelerator, which makes focus jump to the labels buddy.

### CODE REVIEW

File: gui/qt/main_window.py [hunk 1]
```diff
@@ -912,19 +912,24 @@ def create_receive_tab(self):
         self.receive_address_e.addCopyButton(self.app)
         self.receive_address_e.setReadOnly(True)
         msg = _('Bitcoin Cash address where the payment should be received. Note that each payment request uses a different Bitcoin Cash address.')
-        self.receive_address_label = HelpLabel(_('Receiving address'), msg)
```
Comment (cculianu) [inline gui/qt/main_window.py]: Heh. Yeah this isn't referred to ever again. Might as well make it go away from a property.

**Comment by cculianu** [comment]
Looks good. Strangely on macOS (which is what I run) I don't believe these work for reasons that are mysterious to me.  Perhaps it goes against Apple UI guidelines or something so Qt, for macOS, disables the Alt+accelerator.

**Comment by EchterAgo** [comment]
Yea, IIRC it is something related to UI guidelines.

**Comment by cculianu** [comment]
Ah. That explains it.  FWIW back when I ran Linux & Windows more I used to be sure to add accelerators for everything as I enjoyed seeing that little `_` underline appear.. it gave me satisfaction.  Being on macOS and that going away -- I forgot about it, ha ha.

**Pull request merged**